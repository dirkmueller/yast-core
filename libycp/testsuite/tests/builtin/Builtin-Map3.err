Parsed:
----------------------------------------------------------------------
add ($[`a:17, `b:11], `b, nil)
----------------------------------------------------------------------
[Interpreter] tests/builtin/Builtin-Map3.ycp:10 Argument (nil) to add(...) is nil
Parsed:
----------------------------------------------------------------------
"** size **"
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
size ($[])
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
size ($[1:"a", 2:"b"])
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
"** remove **"
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
remove ($[1:2], 0)
----------------------------------------------------------------------
[Interpreter] tests/builtin/Builtin-Map3.ycp:20 Key 0 for remove () does not exist
Parsed:
----------------------------------------------------------------------
remove ($[1:2], 1)
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
remove ($[1:2, 3:4], 1)
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
"** change **"
----------------------------------------------------------------------
[Parser] tests/builtin/Builtin-Map3.ycp:27 Warning: change(...) is deprecated, please fix
Parsed:
----------------------------------------------------------------------
change ($["a":true], "a", false)
----------------------------------------------------------------------
[libycp] tests/builtin/Builtin-Map3.ycp:27 Change does not work as expected! The argument is not passed by reference.
