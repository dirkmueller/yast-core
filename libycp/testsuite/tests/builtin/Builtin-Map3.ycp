
# ---------------------------------------------------------
#
#  Filename:	Builtin-Map.ycp
#
#  Purpose:	test cases for all map builtins
#
# ---------------------------------------------------------

(add ($[`a: 17, `b: 11], `b, nil))


("** size **")

(size ($[]))
(size ($[1:"a", 2:"b"]))

("** remove **")

(remove($[1:2], 0))
(remove($[1:2], 1))
(remove ($[1:2, 3:4], 1))

("** change **")

# deprecated
(change ($["a":true], "a", false))
