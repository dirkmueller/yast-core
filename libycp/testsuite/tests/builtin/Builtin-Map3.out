(nil)
("** size **")
(0)
(2)
("** remove **")
(nil)
($[])
($[3:4])
("** change **")
($["a":false])
